OMGVA PoS/Business layer/Controllers/TaxController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using OMGVA_PoS.Data_layer.DTOs;
+using OMGVA_PoS.Data_layer.Mappers;
+using OMGVA_PoS.Data_layer.Repositories.Tax;
+
+namespace OMGVA_PoS.Business_layer.Controllers
+{
+    [ApiController]
+    [Route("tax")]
+    public class TaxController(ITaxRepository taxRepository) : ControllerBase
+    {
+
+        private static string TaxNotFound(long id) => $"Tax with ID {id} not found.";
```

Comment:
Isn't this too descriptive?

Reply:
It is, you're right. I can make this error message more abstract and log the id if needed